src/Models/board.ts

```diff
@@ -0,0 +1,63 @@
+import mongoose from 'mongoose';
+
+const listSubSchema = new mongoose.Schema({
+  name: {
+    type: String,
+    required: true,
+  },
+  level: {
+    type: Number,
+    required: true,
+  },
+});
+
+const defaultList = [
+  {
+    name: 'Todo',
+    level: 0,
+  },
+  {
+    name: 'In progress',
+    level: 1,
+  },
+  {
+    name: 'Done',
+    level: 2,
+  },
+];
+
+const boardSchema = new mongoose.Schema({
+  title: {
+    type: String,
+    required: true,
+  },
+  isArchived: {
+    type: Boolean,
+    default: false,
+  },
+  lists: {
+    type: [listSubSchema],
+    default: defaultList,
+  },
+  cards: [
+    {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: 'card',
+    },
+  ],
+  admin: {
+    type: mongoose.Schema.Types.ObjectId,
+    ref: 'user',
+  },
+  members: [
+    {
+      type: mongoose.Schema.Types.ObjectId,
+      ref: 'user',
+    },
+  ],
+});
+
+const Board = mongoose.model('board', boardSchema);
+
+export const List = mongoose.model('list', listSubSchema);
+export default Board;
```

Comment:
For better readability. Separate different models in a different files such that you don't need to find where any model is. You tend to forget things one your project becomes huge.